testing/sdk_marathon.py

```diff
@@ -88,6 +88,22 @@ def wait_for_response():
     return config
 
 
+def is_app_running(app: dict) -> bool:
+    return app['tasksStaged'] == 0 and app['tasksUnhealthy'] == 0 and app['tasksRunning'] > 0
+
+
+def wait_for_deployment_and_app_running(app_name: str, timeout: int):
+    shakedown.deployment_wait(timeout, app_name)
+
+    def app_running():
+        cmd = 'marathon app show {}'.format(app_name)
+        log.info('Running %s', cmd)
+        app = sdk_cmd.get_json_output(cmd)
+        return is_app_running(app)
+
+    shakedown.time_wait(app_running, timeout_seconds=timeout)
```

Comment:
I would love to use `retrying` instead, but since we already rely on `shakedown` here, I'm happy to leave this as is.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2528/commits/18f68c6cb4bf3a4708effdadf5c1ad8c0ed010d6